truss/truss/config.go

```diff
@@ -0,0 +1,41 @@
+package truss
+
+import (
+	"path/filepath"
+)
+
+// Config defines the inputs to a truss service generation
+type Config struct {
+	// The first path in $GOPATH
+	GOPATH string
+
+	// The path to where .pb.go files protoc-gen-go creates will be written
```

Comment:
It always picks the first path in the gopath like the go tooling.
```
	// GOPATH
	goPaths := filepath.SplitList(os.Getenv("GOPATH"))
	if goPaths == nil {
		return nil, errors.New("$GOPATH not set")
	}
	cfg.GOPATH = goPaths[0]
```
https://github.com/hasAdamr/go-truss/blob/e765fa9d0e4889468a088fac5710aa19205d7383/truss/main.go#L67